vignettes/pr_review_guidance.Rmd

```diff
@@ -0,0 +1,79 @@
+---
+title: "Pull Request Review Guidance"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{pr_review_guidance}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Fo a pull request (PR) to be merged it needs to pass the automated `R CMD check` and `lintr` workflows at a minimum. In addition the PR reviewer should make sure that
+
+Make sure that
+
+- the function header properly explains the function
+
+- documentation is created/updated by running `devtools::document()`
+
+- functions which are supposed to be exported are listed in the `NAMESPACE` file
+
+- examples print source and newly created variables in their output
```

Comment:
I would update: examples print relevant source variables and newly created variables/record in their output

Reply:
I like that. Thanks!